### PR TITLE
recycler view added

### PR DESCRIPTION
**Pull request by manya-19**

### CODE REVIEW

File: Locationapp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.iml
```
Comment (kartikarora) [inline Locationapp/.gitignore:1]: Consider adding the entire `.idea` folder to .gitignore

Then run

```git
git rm -r --cached *
git add --all
git commit -m <commit_message>
git push origin master
```

File: Locationapp/app/src/main/java/com/example/manya/locationapp/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.example.manya.locationapp;
+
+import android.support.v7.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+public class MainActivity extends AppCompatActivity {
+    private PlacesListAdapter Adapter;
```
Comment (kartikarora) [inline Locationapp/app/src/main/java/com/example/manya/locationapp/MainActivity.java:9]: Readup [this](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md) for naming and code guidelines.

**Review by kartikarora — COMMENTED** [review]
@sakethkaparthi I've added a few comments on the commit,if you don't mind? I can remove them if you have an issue!

**Review by sakethkaparthi — CHANGES_REQUESTED** [review]
@manya-19 please consider the things mentioned above by @kartikarora and make the changes

**Review by sakethkaparthi — APPROVED** [review]

**Pull request merged**